pkg/label/label.go

```diff
@@ -46,3 +47,28 @@ func CreateLabel(key string, value string, isInternal bool) (*v1.Label, error) {
 	}
 	return lbl, nil
 }
+
+func GetLabels(query string, isInternal bool, size int, conn *sdk.Connection) ([]*v1.Label, error) {
+	search := query
+	if isInternal {
+		search += " and internal=true"
+	}
+	labels, err := conn.AccountsMgmt().V1().Labels().List().Size(size).Search(search).Send()
+	if err != nil {
+		return []*v1.Label{}, fmt.Errorf("can't retrieve labels: %w", err)
+	}
+	return labels.Items().Slice(), nil
+}
+
+func GetLabel(id string, conn *sdk.Connection) (*v1.Label, error) {
+	search := fmt.Sprint("id = '", id, "'")
+	lblResponse, err := conn.AccountsMgmt().V1().Labels().List().Search(search).Send()
+	if err != nil {
+		return nil, fmt.Errorf("can't retrieve label: %w", err)
+	}
+	lbl := lblResponse.Items().Slice()
+	if len(lbl) == 0 {
+		return nil, fmt.Errorf("label with id %s not found", id)
+	}
```

Comment:
since the lbl var is a slice (holds multiple values), consider changing it to plural.

Reply:
Changed to labels